install/kubernetes/helm/istio/values-istio-demo-auth.yaml

```diff
@@ -27,3 +27,6 @@ tracing:
 
 servicegraph:
   enabled: true
+
+kiali:
```

Comment:
Please don't enable by default - we don't yet have a process for adding more components, but certainly 'on by default' requires a higher bar.

Reply:
OK, will try to enable it as default when kiali reach 1.0 or a more stable version